scripts/aws/cf-gen.yaml

```diff
@@ -290,6 +290,13 @@ Resources:
           #!/bin/bash -xe
           echo "====== Authorizing public key ======" | tee -a /tmp/init_output.txt
           echo "${PublicKeyString}" >> /home/ec2-user/.ssh/authorized_keys
+          # Redirect /tmp to /var/tmp to avoid tmpfs RAM usage
+          echo "====== Redirecting /tmp to /var/tmp ======" | tee -a /tmp/init_output.txt
+          # Move tmpfs /tmp out of the way and create symlink to disk-backed /var/tmp
+          mv /tmp /tmp.tmpfs
+          ln -sf /var/tmp /tmp
+          echo "====== /tmp successfully redirected to /var/tmp ======" | tee -a /tmp/init_output.txt
```

Comment:
This file is not directly used by the CI jobs until it's merged in the main branch.
Can we test both metal and non-metal AWS instance creation with this new template?

I'd like to make sure the options work as intended. Can we add the output of the following commands to the PR description?
```
lsblk
df -h
ls -ld /tmp /var/tmp
df -h /var/ /tmp/ /var/tmp/
```

Reply:
/hold

Reply:
cc: @ggiguash